### DISCUSSION TITLE
Strange failures with FreeBSD 14.0

### DISCUSSION THREAD

**Discussion by Xottab-DUTY** [cross-platform-actions/action#93] [General]
I have the build failing with FreeBSD 14.0 here: https://github.com/OpenXRay/xray-16/actions/runs/11396244548/job/31709756744#step:3:96

Not sure if it's something related to FreeBSD itself or this action.

I have used '14.0` entry in the .yml as the readme instructs.

**Comment by jacob-carlborg** [comment]
It's FreeBSD. There's a pretty clear error message on lines 87-90 [1]. If you look here [2] you can see that FreeBSD 14.0 is end of life, i.e. not supported anymore. FreeBSD regularly updates the currently supported minor version. What happens here is that the upstream repository for the FreeBSD 14.x branch has been updated from 14.0 to 14.1. That means FreeBSD 14.0 cannot use that repository out of the box. As the error message says, you can either set the `IGNORE_OSVERSION=yes` environment variable to use the 14.1 repository anyway. I think that mostly works, because 14.0 and 14.1 are binary compatible. Or you can use FreeBSD 14.1 instead.

> I have used '14.0` entry in the .yml as the readme instructs.

Well, the examples in the readme are more of examples, to show the syntax. You should use the version you actually want to use, not the exact one specified in the readme. I've just forgot to update the example. The second example actually uses 14.1.

[1] https://github.com/OpenXRay/xray-16/actions/runs/11396244548/job/31709756744#step:3:93
[2] https://www.freebsd.org/releases/

**Comment by Xottab-DUTY** [comment]
Ok, thank you!

Yeah, I switched to 14.1 even before creating this discussion, just after I saw the error :)
